wpeverest/everest-forms
# Enhanced - Improve choice field with image support

### All Submissions:






### Changes proposed in this Pull Request:



In the regular select, radio and checkbox field there exists choice field for options. Now they will support image (but not for select/dropdown which will be later supported).

Closes #111.

### How to test the changes in this Pull Request:

1. Add the choice supported field.
2. Checkout this branch and see if old field are backward compatiable.
3. Now add the images as the options and test if all goes well.
4. That's all we are ready for the image support!

Note: This PR also fixes likert field of survey addon and captcha question-answer field so proceed for those test too.

### Types of changes:




### Features/fix introduced:

* [x] Sortable design issue and smooth choices.
* [x] Limit the number of options to be checked.
* [x] Randomize the order of the choices support.
* [x] Choice default not working as expected on frontend.
* [x] Layout for displaying field choices via style customizer. i.e inline or {one|two|three} columns.

### Other information:




### Changelog entry

> Enhanced - Improve choice field with image support.
